articles/private-saas-deployment/index.md

```diff
@@ -0,0 +1,51 @@
+---
+section: private-saas
+description: Overview of the Private SaaS deployment options
+classes: topic-page
+topics:
+    - private-saas
+    - private-cloud
+    - managed-private-cloud
+contentType: concept
+useCase: private-saas
+---
+<div class="topic-page-header">
+  <div data-name="example" class="topic-page-badge"></div>
+  <h1>Private Saas Deployment Models</h1>
+  <p>
+    A low-friction, dedicated Auth0 deployment that exists in Auth0's Private Cloud.
+  </p>
+</div>
+
+Users with requirements not met by the Auth0 Public Cloud may instead opt for a Private SaaS deployment option.
+
+Auth0 currently offers two Private SaaS deployment models:
+
+* [Private Cloud](/private-saas-deployment/private-cloud)
+* [**Managed** Private Cloud](/private-saas-deployment/managed-private-cloud), either hosted by Auth0 or hosted by you on an AWS environment and operated by Auth0 as a managed service
+
+Private SaaS deployments are single-tenant, isolated instances where none of a customer's resources (software and infrastructure) are shared with any other tenants. This offers increased performance, stability, and availability.
+
+## Private Cloud options and comparison
+
+Auth0 offers two Private Cloud options to those for whom the Public Cloud is not an appropriate deployment option. Here is how the two Private Cloud deployment options compare to each other, as well as how they compare to the Enterprise (Public Cloud) option.
```

Comment:
~Auth0 offers two Private Cloud options to those for whom the Public Cloud is not an appropriate deployment option.~ Here's how the Private Cloud and Enterprise (Public Cloud) deployment options compare to each other.